### ISSUE TITLE
IDE fails to parse `invalid_args_report` message from ViperServer

### ISSUE THREAD

**Issue by JonasAlaif** [viperproject/viper-ide#221]
If a backend is setup with invalid flags then it will (correctly) report an error. This is given to the IDE as a `invalid_args_report` message, which is ignored:
```
Received non-standard ViperServer message of type invalid_args_report.
```
And verification only stops once it times out. This should be improved to instead report the error back to the user and remove the verification in progress bar.

**Comment by JonasAlaif** [comment]
To reproduce this, one can use e.g. `--invalidFlag` in the settings of a backend:
```
...,
"customArguments": "--invalidFlag --z3Exe $z3Exe$ --boogieExe $boogieExe$ $disableCaching$ $fileToVerify$"
```

**Comment by ArquintL** [comment]
should be fixed by #113
